src/main/java/pl/byczazagroda/trackexpensesappbackend/service/UserServiceImpl.java

```diff
@@ -135,4 +136,18 @@ private Cookie createRefreshTokenCookie(User user) {
         return cookie;
     }
 
+    private void validatePasswordLength(String password) {
+
+        if (password.length() < 8) {
```

Comment:
Why still not using contains for "magic value" - `8`?
as well for `100` value below.

Please change it.

Reply:
Updated.